fineract-provider/src/main/java/org/apache/fineract/accounting/journalentry/data/LoanDTO.java

```diff
@@ -45,4 +46,6 @@ public class LoanDTO {
     private boolean markedAsChargeOff;
     @Setter
     private boolean markedAsFraud;
+    @Setter
+    private BigDecimal overpaidAmount;
```

Comment:
Not needed, overpayment amount can be found in the `LoanTransactionDTO`

Reply:
removed it